### ISSUE TITLE
frontend: panic when handling empty query

### ISSUE THREAD

**Issue by yezizp2012** [risingwavelabs/risingwave#6930]
### Describe the bug

```
2022-12-16T06:24:40.887274Z DEBUG risingwave_frontend::scheduler::local: Starting to run query: QueryId { id: "cbf57dcb-3dc8-4293-961d-72e23163a60d" }, sql: ''
2022-12-16T06:24:40.887401Z DEBUG risingwave_frontend::scheduler::local: Local execution mode converts a plan with two stages
thread 'risingwave-main' panicked at 'assertion failed: offset < data_chunk.capacity()', src/common/src/util/chunk_coalesce.rs:253:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/bdb07a8ec8e77aa10fb84fae1d4ff71c21180bb4/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/bdb07a8ec8e77aa10fb84fae1d4ff71c21180bb4/library/core/src/panicking.rs:64:14
   2: core::panicking::panic
```

### To Reproduce

Connect to rw via psql, type `show ` and click tab trying to complete the command, the frontend will panic.
```sql
dev=> show 
```

**Comment by liurenjie1024** [comment]
Not reproducable.

**Comment by yezizp2012** [comment]
Been fixed by #6998 .
